cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/aggregation/NGSIGenericColumnAggregator.java

```diff
@@ -108,7 +108,8 @@ public void aggregate(NGSIEvent event) {
             String attrType = contextAttribute.getType();
             JsonElement attrValue = contextAttribute.getValue();
             String attrMetadata = contextAttribute.getContextMetadata();
-            JsonArray jsonAttrMetadata = new Gson().fromJson(attrMetadata, JsonArray.class);
+            JsonParser jsonParser = new JsonParser();
+            JsonArray jsonAttrMetadata = (JsonArray) jsonParser.parse(attrMetadata);
```

Comment:
Maybe some test should be added covering the failing case (now solved by the PR)?

Reply:
previous PR https://github.com/telefonicaid/fiware-cygnus/pull/1919 were modifying existing tests about this feature

Reply:
So I understand new tests are not needed.

In that case, NTC.